modules/support-gather-data.adoc

```diff
@@ -31,7 +31,7 @@ If this command fails, for example if you cannot schedule a Pod on your cluster,
 +
 [NOTE]
 ====
-If your cluster is using a restricted network you must import the default must-gather image before running the `oc adm must-gather` command.
+If your cluster is using a restricted network you must take additional steps. If you mirror registry has a trusted CA, you must first add the trusted CA to the cluster. For all clusters on restricted networks, you must import the default `must-gather` image as ImageStream before using the `oc adm must-gather` command.
```

Comment:
- Comma after `If your cluster is using a restricted network`
- s/If you mirror registry has/If your mirror registry has
- s/as ImageStream/as an \`ImageStream\`